version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "3.15.0"
+version in ThisBuild := "4.0.0"
```

Comment:
Did we downgrade sbt or something?

Reply:
Huh, I didn't notice that.  That shouldn't have changed back.  :eyes:

Reply:
Actually it doesn't look like this project uses sbt-release at all, so I'm not sure what changed that.